src/scrapers/visa-cal.ts

```diff
@@ -126,6 +136,30 @@ function getTransactionInstallments(memo: string): TransactionInstallments | nul
     total: parseInt(parsedMemo[2], 10),
   };
 }
+
+function getIdentifierAndNumerator(onclickValue: string | null): { identifier?: string, numerator?: string } {
+  if (!onclickValue) {
+    debug('onclick attribute not found for transaction');
```

Comment:
Consider adding explanation what does it mean, something like `cannot extract the identifier of a transaction, ...`

Also, does the debug messages indicate on a bug?

If it can sometimes happen then I recommend not to write them as it will clutter the logs. Lets keep them only if when happens it is a bug that must be fixed.

Reply:
Thanks
Will add more context to the message, as suggested.
I didn't saw this cases when there is no onclick attribute (or other debug cases below), so maybe debug log will help me capture those. wdyt?